cmd/config/docs/api-conventions/functions-impl.md

```diff
@@ -0,0 +1,181 @@
+# Running Configuration Functions using kustomize CLI
+
+Configuration functions as defined [Configuration Functions Specification](functions-spec.md)
```

Comment:
links such as this one should probably include the command to pull up that document -- e.g. `kustomize help config docs-function-spec` or something

Reply:
Done.